packages/consumption/src/modules/notifications/NotificationsController.ts

```diff
@@ -33,8 +33,8 @@ export class NotificationsController extends ConsumptionBaseController {
         return LocalNotification.from(notification);
     }
 
-    public async sent(message: Message): Promise<LocalNotification> {
-        if (!message.isOwn) throw new Error("Cannot send a Notification from a foreign message.");
+    public async saveSent(message: Message): Promise<LocalNotification> {
+        if (!message.isOwn) throw new Error("Cannot save a LocalNotification from a foreign message.");
```

Comment:
I would write 'Cannot save a sent LocalNotification from a received Message.', because in this case we create a LocalNotification with LocalNotificationStatus.Sent from an own Message and  in the received method there will be created a LocalNotification with LocalNotificationStatus.Open and receivedByDevice is not undefined. 
So I wouldn't adjust the received method.